packages/database/tests/pluto.test.ts

```diff
@@ -78,14 +78,13 @@ describe("Pluto encrypted testing with different storages", () => {
   let currentDBName: string;
 
   afterEach(async () => {
-    jest.useRealTimers();
+    // jest.useRealTimers();
     sandbox.restore();
   });
 
   beforeEach(async () => {
-    jest.useFakeTimers();
+    // jest.useFakeTimers();
```

Comment:
:fire: I don't know what these were... but we don't have `jest` any more. Guess we should rm

Reply:
its ok, we just had this in case we needed to test things in time, but i really don't see use case atm.